client/components/thank-you-v2/footer/index.tsx

```diff
@@ -12,24 +14,43 @@ export type ThankYouFooterDetailProps = {
 	buttonText?: string;
 	buttonHref?: string;
 	buttonOnClick?: () => void;
+	supportDoc?: { url: string; id: number };
+	showHelpCenterOnClick?: boolean;
 };
 
+const HELP_CENTER_STORE = HelpCenter.register();
+
 const ThankYouFooterDetail = ( {
 	title,
 	description,
 	buttonText,
 	buttonHref,
 	buttonOnClick,
+	supportDoc,
+	showHelpCenterOnClick,
 }: ThankYouFooterDetailProps ) => {
 	let button = null;
 
-	if ( buttonText && ( buttonHref || buttonOnClick ) ) {
+	const { setShowHelpCenter, setShowSupportDoc } = useDispatch( HELP_CENTER_STORE );
+	const SUPPORT_SITE_ID = 9619154;
+
+	const onClick = () => {
+		if ( supportDoc ) {
+			setShowSupportDoc( supportDoc.url, supportDoc.id, SUPPORT_SITE_ID );
+		} else if ( showHelpCenterOnClick ) {
+			setShowHelpCenter( true );
+		}
+
+		buttonOnClick?.();
+	};
+
+	if ( buttonText && ( buttonHref || buttonOnClick || supportDoc ) ) {
```

Comment:
Hm, if I understand correctly - it won't work correctly when we have `showHelpCenterOnClick` as `true`?

```suggestion
	if ( buttonText && ( buttonHref || buttonOnClick || supportDoc || showHelpCenterOnClick) ) {
```

Reply:
🤦 yep, that'd help, wouldn't it?

Change applied. I'm still having trouble with those two non-help-center links, but I'm 99% sure it's a local issue. It happens if I switch to `trunk` too. Going to reboot the laptop, that usually fixes pretty much anything 😆

Reply:
yep, that did it. all links working well locally now.